app/config/FeatureSwitches.scala

```diff
@@ -19,17 +19,16 @@ package config
 import play.api.Configuration
 import routing.Version
 
-case class FeatureSwitch(maybeConfig: Option[Configuration]) {
+case class FeatureSwitches(featureSwitchConfig: Configuration) {
 
   def isVersionEnabled(version: Version): Boolean =
     (for {
-      config  <- maybeConfig
-      enabled <- config.getOptional[Boolean](s"version-${version.configName}.enabled")
+      enabled <- featureSwitchConfig.getOptional[Boolean](s"version-${version.configName}.enabled")
     } yield enabled).getOrElse(false)
 
   def isAmendLossClaimsOrderRouteEnabled: Boolean =
-    maybeConfig match {
-      case Some(configuration) => configuration.getOptional[Boolean]("amend-loss-claim-order.enabled").getOrElse(false)
-      case None                => false
-    }
+      featureSwitchConfig.getOptional[Boolean]("amend-loss-claim-order.enabled").getOrElse(false)
+
+  val isTaxYearSpecificApiEnabled: Boolean = isEnabled("tys-api.enabled")
+  private def isEnabled(key: String): Boolean = featureSwitchConfig.getOptional[Boolean](key).getOrElse(true)
```

Comment:
can there be a blank line between lines 32 & 33

Reply:
Fixed